modules/alloy/src/main/ts/ephox/alloy/behaviour/highlighting/HighlightApis.ts

```diff
@@ -70,25 +70,17 @@ const highlightBy = (component: AlloyComponent, hConfig: HighlightingConfig, hSt
   });
 };
 
-const isHighlighted = (component: AlloyComponent, hConfig: HighlightingConfig, hState: Stateless, queryTarget: AlloyComponent): boolean => {
-  return Class.has(queryTarget.element(), hConfig.highlightClass);
-};
+const isHighlighted = (component: AlloyComponent, hConfig: HighlightingConfig, hState: Stateless, queryTarget: AlloyComponent): boolean => Class.has(queryTarget.element(), hConfig.highlightClass);
 
-const getHighlighted = (component: AlloyComponent, hConfig: HighlightingConfig, _hState: Stateless): Option<AlloyComponent> => {
-  return SelectorFind.descendant(component.element(), '.' + hConfig.highlightClass).bind((e) => component.getSystem().getByDom(e).toOption());
-};
+const getHighlighted = (component: AlloyComponent, hConfig: HighlightingConfig, _hState: Stateless): Option<AlloyComponent> => SelectorFind.descendant(component.element(), '.' + hConfig.highlightClass).bind((e) => component.getSystem().getByDom(e).toOption());
```

Comment:
Some more (3-4) in this file where it's been shoved on a single line making it hard to read.

Reply:
Fixed